packages/renderer/src/lib/ui/Dropdown.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+export let id: string | undefined = undefined;
+export let name: string | undefined = undefined;
+export let value: unknown;
+export let readonly: boolean = false;
+export let disabled: boolean = false;
+export let required: boolean = false;
+export let error: boolean = false;
+
+let enabled: boolean = true;
+$: enabled = !readonly && !disabled;
+</script>
+
+<select
```

Comment:
IMO We should not use `<select>` as it is a _bad_[^1] html element, with very hard customization, limiting strongly its usage :(


[^1]: This is opinionated, based on experience on many projects where select was always annoying, and the result was always using a custom component, ([react-select](https://react-select.com/home), [svelte-select](https://github.com/rob-balfre/svelte-select), [vue-select](https://github.com/sagalbot/vue-select) etc.) all having for the react 4M download/week.